username_api.py

```diff
@@ -16,15 +19,40 @@ def check_username(website, username):
 	'behance'   :'https://{}.net/{}'.format(website, username)
 	}.get(website, 'https://{}.com/{}'.format(website, username)) # default
 
+	possible = check_format(website, username)
+
+	if not possible:
+		return {
+			'url': url,
+			'possible': possible,
+		}
+
 	if website in ['pinterest', 'gitlab']:
 		res  = r.get(url)
 		code = 200 if bytes(username, encoding='utf-8') in res.content \
 			else 404
 
-		return {'status': code, 'url': url}
-	
+		return {
+			'status': code,
+			'url': url,
+			'possible': possible,
+		}
+	else:
+		return {
+			'status': r.get(url).status_code,
+			'url': url,
+			'possible': possible,
+		}
+
+def check_format(website, username):
```

Comment:
can you add a comment here describing what this function does?

Reply:
(ftr, this was done, but github review comment is pinned to a visible line.)